### ISSUE TITLE
[question] convert to guppy models

### ISSUE THREAD

**Issue by JWDebler** [nanoporetech/bonito#151]
Hi,
I am not sure if this is a thing, but from what I've read it sounded like you can't just use the bonito model files with guppy, but they needed to be converted somehow. Is that correct?
If so, is there a tool that can do it, so we don't have to wait for the compatible models to show up in the rerio repo?

Cheers

**Comment by iiSeymour** [comment]
Yes, that is correct, you can export the model with bonito - 

```
$ bonito export /data/models/my-models > /data/models/my-models/weights.jsn
$ guppy_basecaller -m /data/models/my-models/weights.jsn ...
```

_Note: Full support for Bonito CRF models will be available from Guppy 5.0 onwards._

**Comment by JWDebler** [comment]
Awesome! Thanks.

**Comment by JWDebler** [comment]
I just ran guppy 4.5.2 with the model r9.4.1@3.3. It seems to have worked, but guppy put all the basecalls into the 'fail' folder. Is that expected?

**Comment by iiSeymour** [comment]
This will be because the qscore calibration parameters in the config you used do not match the model so all the scores will be artificially low. The actual calls should be good, you can turn off/lower the qscore filtering in guppy for now but this won't be an issue when the models are officially released in guppy 5.0+.
